src/silo/database.cpp

```diff
@@ -514,6 +512,9 @@ Database Database::preprocessing(
    Database database;
    database.database_config = database_config_;
 
+   SPDLOG_INFO("preprocessing - mining data version");
+   database.setDataVersion(DataVersion(DataVersion::mineDataVersion()));
```

Comment:
I think I actually find it a bit weird, that `DataVersion::mineDataVersion()` doesn't already return a `DataVersion`